superset/translations/fr/LC_MESSAGES/messages.json

```diff
@@ -1635,7 +1643,10 @@
       "Export": ["Exporter"],
       "Export dashboards?": ["Exporter les tableaux de bords ?"],
       "Export query": ["Exporter la requête"],
-      "Export to YAML": ["Exporter en YAML"],
+      "Export to .CSV": ["Exporter au format CSV"],
+      "Export to .JSON": ["Exporter au format YAML"],
```

Comment:
Yikes I missed this one - it should say JSON

Reply:
Oh yes sure, my bad